### PR TITLE
Split long words to fit in the line width limit

### PR DESCRIPTION
**Pull request by diogojs**
Close issue #17

### CODE REVIEW

**Review by GabrielSimonetto — APPROVED** [review]
This is an incredible contribution that will make the world a better place for years to come!

File: picobot/painter.py [hunk 1]
```diff
@@ -205,7 +205,13 @@ def sticker_from_text(
...
+    splitted = text.split(' ')
+    words = []
+    for word in splitted:
+        words.extend(
+            [word[i : i + line_limit] for i in range(0, len(word), line_limit)]
+        )
+
     lines = [words[0]]
     for w in words[1:]:
         if len(lines[-1]) + len(w) < line_limit:
```
Comment (JPTIZ) [inline picobot/painter.py:217]: Doesn't [`textwrap.wrap`](https://docs.python.org/3/library/textwrap.html#textwrap.wrap) already solve this? As in:

```python
def wrapped_text(text: str, line_limit=25):
    return textwrap.wrap(text, width=line_limit)
```

Comment (diogojs) [reply picobot/painter.py:217]: :man_facepalming:

Comment (GabrielSimonetto) [reply picobot/painter.py:217]: I cry evetiem

**Review by aBARICHELLO — APPROVED** [review]

File: picobot/painter.py [hunk 2]
```diff
@@ -262,31 +273,34 @@ def sticker_from_text(
...
+def wrapped_text(text: str, line_width=25, max_lines=None):
+    return '\n'.join(textwrap.wrap(text, width=line_width, max_lines=max_lines))
 
 
 def sticker_from_image(jpg_path: Path):
     img: Image = Image.open(jpg_path)
+    img = resize_to_sticker_limits(img)
+    img_path = jpg_path.with_suffix('.png')
+    img.save(img_path)
+    img.close()
```
Comment (JPTIZ) [inline picobot/painter.py:292]: We can ensure the `img.close` using an `with` statement:

```suggestion
    with Image.open(jpg_path) as img:
        img = resize_to_sticker_limits(img)
        img_path = jpg_path.with_suffix('.png')
        img.save(img_path)
```

File: picobot/painter.py [hunk 3]
```diff
@@ -222,9 +224,18 @@ def sticker_from_text(
...
         bigger_size = title_size
     else:
-        if len(text) >= 26:
-            aux_text = wrapped_text(text, line_limit=26)
+        if len(text) >= LINE_WIDTH_LIMIT:
+            line_width = LINE_WIDTH_LIMIT
+            aux_text = wrapped_text(text, line_width=line_width)
             text_size = aux_img.multiline_textsize(aux_text, font=font)
+            # Try to make a better width/height ratio for long messages
+            for i in range(3):
```
Comment (JPTIZ) [inline picobot/painter.py:232]: Small nit: as `i` is not being used, you can leave it as an `_`:

```suggestion
            for _ in range(3):
```

**Review by JPTIZ — APPROVED** [review]

**Pull request merged**